packages/hub/test/scope.test.ts

```diff
@@ -511,10 +511,10 @@ describe('Scope', () => {
         contexts: { bar: { id: '3' }, baz: { id: '4' } },
         extra: { bar: '3', baz: '4' },
         fingerprint: ['bar'],
-        level: 'warning',
+        level: 'warning' as const,
```

Comment:
why do we need the `const` cast? Because of `SeverityLevel`? Hmm this might make upgrading more tricky for users then right?

Reply:
Yeah, I had that same thought. And yet, here are just a few of the many examples of other places in the SDK, both in source code and tests, where this wasn't necessary:

![image](https://user-images.githubusercontent.com/14812505/163467768-06700c88-fe97-4177-ba8e-d58fb55819c5.png)

This PR is basically a collection of all of the fixes I've had to make ever since I started work on upgrading jest, and it's been rebased any number of times - the PRs on which it depends had to work their way though to getting merged, and in the meantime things in the repo have been changing rapidly and I've kept having to add new fixes. So it's possible that that was necessary at some stage along the way and now isn't, maybe? Because I don't see how this instance is different from all of the ones above. I'll try again without.